### PR TITLE
Felinid mood fix

### PR DESCRIPTION
**Pull request by mamiipolat**
resim

## About The Pull Request
Showering will add negative mood effect to felinids
## Why It's Good For The Game
there was a status effect that drops stamina for felinids on shower but at the same time it boosts mood i decided to make it better
:cl: Mamaii
add: shower will give felinids negative mood effect
fix: fixed shower hater status effect alert not showing 
/:cl:

### CODE REVIEW

File: code/datums/mood_events/needs_events.dm [hunk 1]
```diff
@@ -76,6 +76,11 @@
 	mood_change = 4
 	timeout = 5 MINUTES
 
+/datum/mood_event/shower_hater
+	description = "I hate being wet!"
+	mood_change = -4
+	timeout = 6 MINUTES
```
Comment (Sadboysuss) [inline code/datums/mood_events/needs_events.dm:82]: 6 minutes is too much

File: code/datums/mood_events/needs_events.dm [hunk 2]
```diff
@@ -76,6 +76,11 @@
 	mood_change = 4
 	timeout = 5 MINUTES
 
+/datum/mood_event/shower_hater
+	description = "I hate being wet!"
+	mood_change = -4
```
Comment (Sadboysuss) [inline code/datums/mood_events/needs_events.dm:81]: -4 is too much

File: code/datums/mood_events/needs_events.dm (hunk 1 above)
Comment (mamiipolat) [reply code/datums/mood_events/needs_events.dm:82]: positive event was 5

File: code/datums/mood_events/needs_events.dm (hunk 2 above)
Comment (mamiipolat) [reply code/datums/mood_events/needs_events.dm:81]: positive one was +4

**Comment by Striders13** [comment]
the status effect seems broken if you wanna fix it also

**Comment by ZephyrTFA** [comment]
This should be a flag on the species datum and not a snowflake isfelinid check

File: code/game/objects/structures/shower.dm [hunk 3]
```diff
@@ -259,8 +259,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/shower, (-16))
 		return
 	var/mob/living/living_target = target
 	check_heat(living_target)
-	living_target.add_mood_event("shower", /datum/mood_event/nice_shower)
 	living_target.apply_status_effect(/datum/status_effect/shower_regen)
+	if(!isfelinid(target))
```
Comment (Ghommie) [inline code/game/objects/structures/shower.dm:263]:
```suggestion
	if(!HAS_TRAIT(target, TRAIT_WATER_HATER) || HAS_TRAIT(target, TRAIT_WATER_ADAPTATION))
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
No snowflake species checks when we have equivalent traits.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**